firmware/serial_debugger/board_protos/VC.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+/*
+For enums, the default value is the first defined enum value, which must be 0.
+For numeric types, the default value is zero.
+*/
+
+enum GpioNetName {
+  GPIO_NET_NAME_UNSPECIFIED  = 0;
+  BUZZER_PWR_EN = 1;
+  BAT_I_SNS_NFLT = 2;
+  LED = 3;
+  TELEM_PWR_EN = 4;
+  NPCM_EN = 5; 
+  ACC_I_SENSE_NFLT = 6;
+  PGOOD = 7;
+  LV_PWR_EN = 8; 
+  AUX_PWR_EN = 9;
+  PUMP_PWR_EN = 10;
+  _900K_GPIO = 11; 
```

Comment:
for any special naming conventions that you and Joe landed on (ie. ! becomes n), please define them in a comment at the top of the file so people adding more pins can follow the same convention